external/dynamo/generalized/router.py

```
@@ -0,0 +1,1039 @@
+
+    # Feedback timeout & sweep
+    parser.add_argument("--feedback-timeout-seconds", type=float, default=120.0)
+    parser.add_argument("--pending-sweep-interval-seconds", type=float, default=5.0)
+    parser.add_argument("--timeout-reward", type=float, default=0.0)
+
+    # Latency EMAThe
+    parser.add_argument("--latency-ema-alpha", type=float, default=0.2)
+
+    # Traces
+    parser.add_argument("--debug-traces", action="store_true", default=False)
+    parser.add_argument("--debug-trace-dir", type=str, default="/tmp/dynamo_router_traces")
```

Ruff (0.14.8)

Probable insecure usage of temporary file or directory: "/tmp/dynamo_router_traces"

(S108)